Error in compiling using dockerfile

make[1]: Leaving directory '/wallet-core/build/local/src/protobuf/protobuf-3.9.0/src'
libprotoc 3.9.0
-- The C compiler identification is Clang 9.0.0
-- The CXX compiler identification is Clang 9.0.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/clang-9 - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/clang++-9 - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
CMake Error at CMakeLists.txt:16 (find_library):
  Could not find Protobuf using the following names:


-- Configuring incomplete, errors occurred!
See also "/wallet-core/protobuf-plugin/build/CMakeFiles/CMakeOutput.log".
The command '/bin/bash -o pipefail -c tools/install-dependencies' returned a non-zero code: 1

--- a/protobuf-plugin/CMakeLists.txt
+++ b/protobuf-plugin/CMakeLists.txt
@@ -13,7 +13,7 @@ endif()
 include_directories(${PREFIX}/include)
 link_directories(${PREFIX}/lib)
 
-find_library(Protobuf REQUIRED PATH ${PREFIX}/lib/pkgconfig)
+find_package(Protobuf REQUIRED PATH ${PREFIX}/lib/pkgconfig)
 include_directories(${Protobuf_INCLUDE_DIRS})
 include_directories(${CMAKE_CURRENT_BINARY_DIR})
 

